hive_metastore_client/builders/serde_info_builder.py

```diff
@@ -34,7 +34,7 @@ def __init__(
         """
         self.name = name
         self.serialization_lib = serialization_lib
-        self.parameters = parameters
+        self.parameters = parameters if parameters else {}
```

Comment:
I would compare it directly to None to avoid overwriting falsy parameters [check this SO reply](https://stackoverflow.com/a/366446/6723594).
Just for information, there is an equivalent form as the one proposed `self.parameters = parameters or {}` which some people consider more readable. But I still think it's worth using the `{} if parameters is None else parameter`

Reply:
Hmm I'm afraid I didn't get what you suggested Breno. Because python considers None as False in boolean comparisons.
Also if we restrict the comparison to `if parameter is None`, we will miss other False values as False, '', 0.

Reply:
Yes, I think it would be better to not overwrite falsy values like False, '' or 0 because it would silently change the values.  
If we want to restrict arguments like 0 and '' we can validate the type and raise an exception for the invalid ones but I don't like the idea of mixing this validation with the default argument behaviour.

Reply:
I tried using other false values for those attributes, and the thrift returns the following error:

![image](https://user-images.githubusercontent.com/31676903/129236877-b3a61ada-f09b-488a-96fb-eb31bcc40976.png)

For this case, we can apply the changes suggested by Breno and change only the attribute when they are None.